package/pyproject.toml

```diff
@@ -11,8 +11,8 @@ requires = [
     # Scipy: On windows avoid 1.21.6, 1.22.0, and 1.22.1 because they were built on vc142
     # Let's set everything to this version to make things clean, also avoids other issues
     # on other archs
-    "numpy==1.22.3; python_version=='3.9' and platform_python_implementation != 'PyPy'",
-    "numpy==1.22.3; python_version=='3.10' and platform_python_implementation != 'PyPy'",
+    "numpy==1.23.2; python_version=='3.9' and platform_python_implementation != 'PyPy'",
+    "numpy==1.23.2; python_version=='3.10' and platform_python_implementation != 'PyPy'",
```

Comment:
FWIW, I think this simplification is safe to do now (but also fine to delay it till later):

```diff
--- a/package/pyproject.toml
+++ b/package/pyproject.toml
@@ -11,9 +11,7 @@ requires = [
     # Scipy: On windows avoid 1.21.6, 1.22.0, and 1.22.1 because they were built on vc142
     # Let's set everything to this version to make things clean, also avoids other issues
     # on other archs
-    "numpy==1.23.2; python_version=='3.9' and platform_python_implementation != 'PyPy'",
-    "numpy==1.23.2; python_version=='3.10' and platform_python_implementation != 'PyPy'",
-    "numpy==1.23.2; python_version=='3.11' and platform_python_implementation != 'PyPy'",
+    "numpy==1.23.2; python_version<='3.11' and platform_python_implementation != 'PyPy'",
     "numpy==1.26.0; python_version=='3.12' and platform_python_implementation != 'PyPy'",
     # For unreleased versions of Python there is currently no known supported
     # NumPy version. In that case we just let it be a bare NumPy install
```

We can probably purge out some of the comments above now too, but also fine to delay that cleanup if you want of course.

Reply:
Ah yeah thanks 😅 I was being rather lazy here, let's go ahead with cleaning this up :)